### DISCUSSION TITLE
Link to documentation?

### DISCUSSION THREAD

**Discussion by sti-yvi** [provectus/kafka-ui#3583] [General]
Hello,

For a new user, it might be hard to find the documentation.
For example, I found these:
- https://github.com/provectus/kafka-ui/tree/master/documentation
- https://github.com/provectus/kafka-ui/blob/master/docker-compose.md
- https://github.com/provectus/kafka-ui/blob/master/helm_chart.md
- https://github.com/provectus/kafka-ui/wiki
- https://github.com/provectus/kafka-ui/tree/docs (GitBook at docs.kafka-ui.provectus.io)

I see some duplicate between the Wiki and the GitBook. The Wiki is probably not up-to-date. Which one is the correct one?

Putting documentation for end users in a single place can be a nice improvement IMHO.

**Comment by Haarolean** [comment]
Hi again,

thanks for pointing this out.

I'm well aware that we have messed up docs, but I didn't get a chance to do a proper refactoring yet.
Just did it, all docs could be found [here ](https://docs.kafka-ui.provectus.io/). 
Wiki got nuked, README got updated, and docs-related md files have been dropped from the repo as well.
Let me know if there are any broken links left and/or if there's anything we can improve regarding docs.
